client/js/utils.js

```diff
@@ -23,6 +23,20 @@ module.exports = {
 	isOpInChannel,
 };
 
+const commands = [
+	"collapse",
+	"expand",
+	"join",
+].reduce(function(cmds, name) {
```

Comment:
Perhaps this could be automated like templates are? https://github.com/thelounge/lounge/blob/master/client/views/index.js